src/service/location/location.service.ts

```diff
@@ -77,7 +77,15 @@ export class LocationService {
     address: string;
     protocol: string;
   } {
-    const multiaddrInstance = new Multiaddr(Buffer.from(multiAddr, 'base64'));
+    let multiaddrInstance = new Multiaddr(Buffer.from(multiAddr, 'base64'));
+
+    // TODO: temporary fix needed because multiaddr library does not support /dns/ prefix
```

Comment:
@kacperzuk-neti there is an issue with `multiaddr` library that I hacked this way. `multiaddr` itself is deprecated and I tried using new `@multiformats/js-multiaddr` but it looks like it's broken too. let me know what do you think

Reply:
New `@multiformats/js-multiaddr` supports `/dns/` and tries to resolve this as dns4 or dns6 afaik. From what I saw, our `/dns/` is always dns4

Reply:
In that case what's the problem with the new `@multiformats/js-multiaddr`?

Reply:
Javascript probably. It breaks the whole build, I wasnt able to add and use this library even in a separate project. We've been working on this with Maciej and it looks like an issue on the library side.

Only one provider is using `/dns/` prefix.

Reply:
Fair enough, let's keep your workaround.